### ISSUE TITLE
Android-Kotlin : 09.22 Calender View in Android (Kotlin)

### ISSUE THREAD

**Issue by sitamadex11** [girlscript/winter-of-contributing#1453]
<hr>

## Description 📜

* Give a step-by-step tutorial of Calender View in Android (Kotlin).
*  Required Documentation or Video or Audio.

<hr>

## Domain of Contribution 📊

- [x] Android Dev (Flutter, Java, Kotlin)

<hr>

<hr>

## References :
> https://www.geeksforgeeks.org/android-creating-a-calendar-view-app/#:~:text=Below%20are%20the%20steps%20for,XML%20file%20and%20java%20file.&text=Step%203%3A%20Now%2C%20open%20up,get%20the%20Calendarview%20and%20textview.

<hr>

## Please Read it Carefully Before Contributing
* Changes should be made in **Android_Development_With_Kotlin/** directory under **Android_Development_With_Kotlin** branch.
* Inside **Android_Development_With_Kotlin/** directory creates a directory and names it according to the topic ( for this issue **09. Dynamic User Interface**)
* Inside **09. Dynamic User Interface/** directory add your task with subtopic number and name and make changes accordingly.
* You have to push your PR to the **Android_Development_With_Kotlin** branch only.
* You can take help to the references but don't simply try to copy it. Just explain in your own word.
* This issue is for **GWOC'21** contributors only.
* You need to push in a single commit. If there exists more than one commit then you need to **squash** your commits into one commit.
* Will assign you on a first come first serve basis.
* Please try to complete your assigned task within 7 days. If you need some more time then, please inform mentors and supervisors.
* If you have two previously assigned issues on **Android-Kotlin** pending then please first complete them.
> Supervisor @rahulsain

## Happy Contributing. Best of Luck.💖

**Comment by Sakalya100** [comment]
I want to work on this issue in the documentation section.
Myself Sakalya Mitra(Batch 3 Android Dev with Kotlin)
@sitamadex11 @rahulsain

**Comment by preeti003** [comment]
@rahulsain is this issue still open for documentation?

**Comment by Sakalya100** [comment]
An update from my end. I am having my exams so the contribution got a bit delayed...I will make a PR in the next 2 days. Apologies for the delay.

**Comment by rahulsain** [comment]
> @rahulsain is this issue still open for documentation?

no

**Comment by sitamadex11** [comment]
@Sakalya100 any update bro?

**Comment by Sakalya100** [comment]
Yes... Actually due to Puja I am a bit busy. I will make the PR soon

**Comment by sitamadex11** [comment]
Sure sure no issues. Take your time.

**Comment by sitamadex11** [comment]
@rahulsain
